docs/content/tutorials/custom-listen-ports.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Customze ports for NGINX Ingress Controller
+description: |
+  Customze ports for NGINX Ingress Controller
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-1191"
+---
+## Customizing the `listen` line in NGINX Ingress Controller.
+
+This document will explain how to change the default ports that NGINX Ingress Controller is configured for, as well as add additional `listen` settings. For more information, please read the [NGINX Listen documentation](http://nginx.org/en/docs/http/ngx_http_core_module.html#listen).
+
+
+## Changing Default Ports
+
+By default, NGINX Ingress Controller listens on ports 80 and 443. These ports can be changed easily, but modifying the `listen` ports for your NGINX Ingress resources will require the editing of .tmpl files.
+
+If you are using `ingress` resource you will need to modify:
+- `nginx-plus-ingress.tmpl` if using NGINX Plus
+- `nginx-ingress.tmpl` if using NGINX OSS
+
+If you are using NGINX Ingress Controller CRDs (virtualServer):
+- `nginx-plus-virtualserver.tmpl` for NGINX Plus
+- `nginx-virtualserver.tmpl` if using NGINX OSS
+
+For this example, we are going to use the `nginx-virtualserver.tmpl` to change the port from 80 to 85.
+Here is a link to the directory for the `.tmpl` files:
+
+[nginx-virtualserver template files](https://github.com/nginxinc/kubernetes-ingress/tree/main/internal/configs/version2)
```

Comment:
The path should also be shown so the reader does not need to click or hover to read the path.